### PR TITLE
fix: Require jwt for accepting group role invite

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes #needed for https://github.com/fossasia/open-event-frontend/pull/7134 

- have to implement role_invite for group.

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by iamareebjamal** [comment]
I meant in frontend, not here

**Comment by iamareebjamal** [comment]
@maze-runnar Test failing

**Comment by maze-runnar** [comment]
> @maze-runnar Test failing

fixing it

**Review by sachinchauhan2889 — COMMENTED** [review]
LGTM!.

only one thing is remaining.

invite link need to change.
/users-groups-roles?token=1234

to 

/group-invites?token=1234

File: app/models/users_groups_role.py [hunk 1]
```diff
@@ -35,7 +35,6 @@ class UsersGroupsRoles(db.Model):
     group_id = db.Column(
         db.Integer, db.ForeignKey('groups.id', ondelete='CASCADE'), nullable=False
     )
-    group = db.relationship("Group", backref='groups_roles')
```
Comment (sachinchauhan2889) [inline app/models/users_groups_role.py:38]: line no 60

users-groups-roles ----> group-invites

Comment (iamareebjamal) [reply app/models/users_groups_role.py:38]: No, both are same now

Comment (sachinchauhan2889) [reply app/models/users_groups_role.py:38]: https://user-images.githubusercontent.com/72552281/115506601-ab717400-a298-11eb-965a-a7fb34d8b9a3.mp4

Comment (sachinchauhan2889) [reply app/models/users_groups_role.py:38]: https://user-images.githubusercontent.com/72552281/115507166-6863d080-a299-11eb-9181-79f61ccf0bef.mp4

**Pull request merged**